lib/CapeMapper/DatasetMapper.js

```diff
@@ -126,9 +126,9 @@ class DatasetMapper {
             }); // end of foreach incoming_record
 
             // note any headings that were never used once we've done the whole table
-            output.unmapped_headings.push(Object.keys(unmapped_headings_in_table));
+            output.unmapped_headings = output.unmapped_headings.concat(Object.keys(unmapped_headings_in_table));
             // ... and any headings we were missing
-            output.missing_headings.push(Object.keys(missing_headings));
+            output.missing_headings = output.missing_headings.concat(Object.keys(missing_headings));
```

Comment:
What's the problem with the old code (with `push(()`)? The old and new codes are doing the same thing (ie provide the same result), but in different ways, right?